app/helpers.go

```diff
@@ -112,10 +113,11 @@ func (s *SliceIterator) Valid() bool {
 // Next moves the iterator to the next sequential key in the database, as
 // defined by order of iteration.
 //
-// If Valid returns false, this method will panic.
-func (s *SliceIterator) Next() {
+// If Valid returns false, this method will panic.keys
```

Comment:
:policeman:

Reply:
Maybe I can fix this... I will look

Reply:
I meant the comment only. Current description makes little sense :)